sdk/confidentialledger/Azure.Security.ConfidentialLedger/tests/samples/HelloWorldSamples.cs

```diff
@@ -44,21 +39,15 @@ public void HelloWorld()
 
             Response statusResponse = ledgerClient.GetTransactionStatus(transactionId);
 
-            string status = JsonDocument.Parse(statusResponse.Content)
-                .RootElement
-                .GetProperty("state")
-                .GetString();
+            string status = statusResponse.Content.ToDynamic().state;
```

Comment:
In an earlier version, we let callers access properties without case sensitivity.  This will need to be added back to `DynamicJson` if we want this feature in the new implementation.